### PR TITLE
A Nightmare on Haag Street

### PR DESCRIPTION
**Pull request by MHaggis**
## Updated Analytics
- Attempt To Delete Services
- Attempt To Disable Services
- Attempted Credential Dump From Registry via Reg exe
- Delete a net user
- Deny Permission using Cacls Utility
- Detect Dump LSASS Memory using comsvcs
- Disable Net User Account
- First time seen command line argument
- Grant Permission Using Cacls Utility
- Prohibited apps spawning cmdprompt
- Potential Pass the Token or Hash Observed at the Destination Device
- Rare Parent-Child Process Relationship
- ptt pth kerb ntlm origin device
- Resize Shadowstorage Volume
- sdelete application execution

## Deprecated BA Analytics

- ssa___applying_stolen_credentials_via_mimikatz_modules.yml
- ssa___applying_stolen_credentials_via_powersploit_modules.yml
- ssa___assess_credential_strength_via_dsinternals_modules.yml
- ssa___credential_extraction_dsinternals_conversion_modules.yml
- Ssa___credential_extraction_dsinternals_modules.yml
- Ssa___credential_extraction_getaddbaccount_from_dump.yml
- Ssa___credential_extraction_powersploit_modules.yml
- ssa___illegal_access_user_content_via_powersploit_modules.yml
- ssa___illegal_account_creation_via_powersploit_modules.yml
- ssa___illegal_account_enable_disable_via_dsinternals_modules.yml
- ssa___illegal_log_deletion_via_mimikatz_modules.yml
- ssa___illegal_management_AD_elements_and_policies_via_dsinternals_modules.yml
- ssa___illegal_management_computers_and_AD_elements_via_powersploit_modules.yml
- Ssa___illegal_privilege_elevation_and_persistence_via_powersploit_modules.yml
- ssa___illegal_privilege_elevation_via_mimikatz_modules.yml
- ssa___illegal_service_and_process_control_via_mimikatz_modules.yml
- ssa___illegal_service_and_process_control_via_powersploit_modules.yml
- Ssa___probing_access_with_stolen_credentials_via_powersploit_modules.yml
- ssa___recon_access_and_persistence_opportunities_via_powersploit_modules.yml
- ssa___recon_and_use_accounts_groups_policies_via_powersploit_modules.yml
- ssa___recon_and_use_accounts_groups_via_mimikatz_modules.yml
- ssa___recon_and_use_active_directory_infrastructure_via_powersploit_modules.yml
- ssa___recon_and_use_computers_domains_via_powersploit_modules.yml
- ssa___recon_and_use_computers_via_mimikatz_modules.yml
- ssa___recon_and_use_operating_system_elements_via_powersploit_modules.yml
- ssa___recon_and_use_shares_via_mimikatz_modules.yml
- ssa___recon_and_use_shares_via_powersploit_modules.yml
- Ssa___recon_connectivity_via_powersploit_modules.yml
- ssa___recon_credential_stores_and_services_via_mimikatz_modules.yml
- ssa___recon_defensive_tools_via_powersploit_modules.yml
- ssa___recon_privilege_escalation_opportunities_via_powersploit_modules.yml
- ssa___recon_process_service_hijacking_via_mimikatz_modules.yml
- ssa___recon_processes_and_services_via_mimikatz_modules.yml
- ssa___setting_credentials_via_dsinternals_modules.yml
- ssa___setting_credentials_via_mimikatz_modules.yml
- ssa___setting_credentials_via_powersploit_modules.yml
- ssa___credential_extraction_fgdump_cachedump_s_option.yml
- Ssa___credential_extraction_fgdump_cachedump_v_option.yml
- ssa___credential_extraction_ms_debuggers_kernel_peek.yml
- ssa___credential_extraction_ms_debuggers_z_option.yml
- Ssa___credential_extraction_lazagne_command_options.yml
-

### CODE REVIEW

File: detections/endpoint/ssa___attempt_to_delete_services.yml [hunk 1]
```diff
@@ -86,3 +85,29 @@ tags:
   risk_score: 36
   risk_severity: high
   security_domain: endpoint
+  impact: 60
```
Comment (josehelps) [inline detections/endpoint/ssa___attempt_to_delete_services.yml]: This seems to have the observable and other tags duplicated

File: detections/endpoint/ssa___attempt_to_disable_services.yml [hunk 2]
```diff
@@ -88,3 +84,29 @@ tags:
   risk_score: 36
   risk_severity: medium
   security_domain: endpoint
+  impact: 60
```
Comment (josehelps) [inline detections/endpoint/ssa___attempt_to_disable_services.yml]: same issue, tags duplicated

File: detections/endpoint/ssa___disable_net_user_account.yml [hunk 3]
```diff
@@ -83,6 +83,30 @@ tags:
   - dest_user_id
   - process
   - cmd_line
-  risk_score: 49
   risk_severity: medium
   security_domain: endpoint
+  impact: 80
```
Comment (josehelps) [inline detections/endpoint/ssa___disable_net_user_account.yml]: tags are duplicated

File: detections/endpoint/ssa___sdelete_application_execution.yml [hunk 4]
```diff
@@ -60,6 +56,28 @@ tags:
   - T1485
   - T1070.004
   - T1070
+  product:
```
Comment (josehelps) [inline detections/endpoint/ssa___sdelete_application_execution.yml]: some tags are duplicated

**Review by josehelps — CHANGES_REQUESTED** [review]
overall looks great, just saw some detections that had duplicated tags.

**Review by patel-bhavin — APPROVED** [review]

**Review by patel-bhavin — APPROVED** [review]

**Review by josehelps — APPROVED** [review]
LGTM

**Pull request merged**